reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -124,10 +125,8 @@ class ReportTableModelMapper(
                 val declaredLicenses = ortResult.getDeclaredLicensesForId(id)
                 val detectedLicenses = licenseFindings[id]?.toSortedMap(compareBy { it.license }) ?: sortedMapOf()
 
-                val analyzerIssues = projectIssues[id].orEmpty().toMutableList()
-                analyzerResult.issues[id]?.let {
-                    analyzerIssues += it
-                }
+                val analyzerIssues =
+                    projectIssues[id].orEmpty() + analyzerResult.issues[id].orEmpty() + analyzerIssuesForPackages[id].orEmpty()
```

Comment:
FYI, the line is still too long. And can we align on the more common (in our code base) formatting to break after the last `+`?

Reply:
Done.